net/get_iplayer/Portfile

```diff
@@ -15,11 +15,11 @@ description         A utility for downloading TV and radio from BBC iPlayer
 long_description    ${description}. \
                     \nThis port does not install the Web PVR Manager (get_iplayer.cgi).
 
-checksums           rmd160  7171e71a1bb1d7426a168956c3312134ba98e8cb \
-                    sha256  9a6f780d616b66f7e890f380f90dc1d8918dc62165847343fe2ac44e95486757 \
-                    size    141452
+checksums           rmd160  011a63971723bb3597ea7c30f4118d7c05616952 \
+                    sha256  0c7721124f424bfc0ba1bbbc5d4965382ef1d473c307c60b2c273e65c9284e65 \
+                    size    146353
 
-perl5.branches      5.26
+# if perl5.branches is not specified the default perl5.major is used (currently 5.28)
```

Comment:
I think there is no need for this comment, the commit message is enough.
However, the commit message, and the comment if you want to retain it, should be precise: the perl5 version actually used can be 5.26 or 5.28, depending on the variant set by the user.
My preference is to remove the comment and fix the commit message.